### PR TITLE
Creates project base and server

### PR DESCRIPTION
**Pull request by Luke-Py-Walker**
### Why

Future endpoints will utilize this server to process requests

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const config =  {
+  development: require('./development'),
+  production: require('./production'),
+  staging: require('./staging'),
+  test: require('./test')
+};
+
+/* istanbul ignore next */
+module.exports = config[process.env.NODE_ENV || 'development'];
```
Comment (nathankhuu) [inline config/index.js]: by the way, looking at the code now, this is where we know which config variable to use

by default, it'll use development, but if you do `NODE_ENV=somethingelse` it'll use the `somethingelse` environment

Comment (Luke-Py-Walker) [reply config/index.js]: Nice catch, I didn't check the index file

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Util = require('util');
+
+const Config = require('../config');
+const Movies = require('./server');
+
+Movies.start(() => {
+  Util.log(`Server started on port: ${Config.PORT}`);
+});
```
Comment (shraddha1704) [inline lib/index.js]: small nit: We usually leave a new line at the end of our files!

**Comment by Luke-Py-Walker** [comment]
> lgtm!
> 
> could you update the PR title / commit message to match our format?
> 
> something like `feat(base): creates project base and server`
> 
> for reference, the format is in this doc:
> 
> https://www.notion.so/lob/Code-Review-Guidelines-aad42d64cb724f3cadd1ab233bfc93df#2057e9b7c83c4cde97a577bc17784b15

The guidelines didn't indicate how the PR title should be formatted, I'll add it now

**Comment by nathankhuu** [comment]
@Luke-Py-Walker yeah the doc is not explicit about it :/ it should just be the same as what they have listed for commit messages

**Pull request merged**